screenpy/actions/see.py

```diff
@@ -50,7 +50,11 @@ def perform_as(self, the_actor: Actor) -> None:
             value = self.question
             aside(f"the actual value is: {value}")
 
-        assert_that(value, self.resolution)
+        reason = ""
```

Comment:
Nice looking to find the base case from `pyhamcrest`, i would have expected `None` here!

Reply:
I just used the same default as`assert_that` sets.  I was afraid to set None for fear of odd behavior.